src/main/resources/api/data-repository-openapi.yaml

```diff
@@ -4432,6 +4432,11 @@ components:
             type: array
             items:
               type: string
+        errors:
```

Comment:
This is better than failing completely but it does mean that a caller now needs to check this value and/or show it to a user.

What should the catalog do in this case, if `errors` is non-empty? Will this API change be removed in the future when the RAS APIs are stable?

Reply:
I think the problem is that we don't quite know yet as the service is maturing.  As a pattern, it's nice to return as part of a successful call but understood that it does put the onus on the call to know to check there.  Initially, it's probably reasonable to just ignore and not show the snapshots that couldn't be verified